src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCF.java

```diff
@@ -277,24 +476,47 @@ private boolean isHomRefBlock(final VariantContext result) {
      * @param result VariantContext to process
      * @return true if VC is a 0/0 call and not a homRef block
      */
-    private boolean isHomRefCall(final VariantContext result) {
+    private boolean isMonomorphicCallWithAlts(final VariantContext result) {
```

Comment:
The `result.getLog10PError() != VariantContext.NO_LOG10_PERROR` is the implicit check for alt alleles because these sites will have a very very low value for QUAL.  And the ./. without homref check is a mistake.

I've fixed both of these (locally).